linkerd/meshtls/Cargo.toml

```diff
@@ -25,6 +25,7 @@ linkerd-meshtls-boring = { path = "boring", optional = true }
 linkerd-meshtls-rustls = { path = "rustls", optional = true }
 linkerd-stack = { path = "../stack" }
 linkerd-tls = { path = "../tls" }
+rcgen = "0.11.3"
```

Comment:
It looks like we're now taking on a new runtime dependency for this test code. That's undesirable. We want to make sure this dependency can be isolated to the dev-dependency graph so it isn't needed in release builds.

I think we should also avoid the circular dependency introduced here, whereby the meshtls/rustls and mesthls/boring tests now depend on the higher-level meshtls crate.

Generally, the way I'd recommend approaching this is to add a new `meshtls/test-util` crate with the helpers in its lib.rs.

Then `meshtls/boring` and `meshtls/rustls` each take dev-dependency on ../test-util.